### PR TITLE
Add Traditional Chinese Translation

### PR DESCRIPTION
**Pull request by GodofMonkeys**
Add Traditional Chinese Translation

### CODE REVIEW

**Comment by oOKexOo** [comment]
Thx for your effort. Looks fine, but we need to test it before the merge.

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 1]
```diff
@@ -5265,25 +5862,36 @@
 			%1&lt;img image='\achilles\data_f_achilles\icons\icon_default_object.paa'/&gt; 该模块可以放置在物体或任何地方。模块给出的选项将视选择而定！
 			%1&lt;img image='\achilles\data_f_achilles\icons\icon_position.paa'/&gt; 该模块会在选择的位置上激活。
 		</Chinesesimp>
+                <Chinese>
+   阿基裡斯模塊的模塊層級中帶有引導性質的圖標%1其中最常見為：
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Weird indentation :5866 - :5873

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 2]
```diff
@@ -5265,25 +5862,36 @@
...
+                <English>
 			For modules that apply to objects or units, it is often the case that you can either place it on one unit or place it elsewhere to apply it to multiple units.
 		</English>
                 <Japanese>
 			物体またはユニットへ適用されるモジュールでは、ユニットに配置するか、別の場所に配置して複数のユニットに適用することができます。
 		</Japanese>
                 <Chinesesimp>可应用于物体或单位的模块一般都可以直接放置在一个单位上，置于其它地方时则可将其应用到多个单位上。
 		</Chinesesimp>
+                <Chinese>可應用於物件或單位的模塊一般都可以直接放置在一個單位上，置於其它地方時則可將其應用到多個單位上。
+  </Chinese>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 3]
```diff
@@ -5327,37 +5936,44 @@
 		</Japanese>
                 <Chinesesimp>
 			部分模块可以将特效应用到被选中的单位上。
-			%1当听到这个声音时你可以选中单位并按下%3[回车键]%4提交选择。
+			%1当听到这个声音时你可以选中单位并按下%3[Enter键]%4提交选择。
 			%1你可以按%3[ESC键]%4终止选择模式。
 		</Chinesesimp>
+                <Chinese>
+   部分模塊可以將特效應用到被選中的單位上。
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 4]
```diff
@@ -5420,13 +6036,24 @@
 			%1%2按下%3[左CTRL]%4 + %3[G]%4来为单位编组。
 			%1%2按下%3[左CTRL]%4 + %3[左SHIFT]%4 + %3[G]%4来取消单位编组。
 		</Chinesesimp>
+                <Chinese>
+   額外鍵位組合：
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 5]
```diff
@@ -5461,10 +6088,16 @@
 		</Japanese>
                 <Chinesesimp>
 			代码执行模块仅能在服务器允许客户端使用的情况下运行！
-			%1%1模块内置了两个params：
+			%1%1模块内置了两个参数：
 			%1%2%3_this select 0%4   位置：模块放置的位置。
 			%1%2%3_this select 1%4   物体：返回ObjNull或模块所属的物体。
 		</Chinesesimp>
+                <Chinese>
+   代碼執行模塊僅能在伺服器允許客戶端使用的情況下運行！
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 6]
```diff
@@ -5529,11 +6165,17 @@
 			%1 3) ユニットのアイコン上で %3[Left Ctrl]%4 + %3[RMB]%4 を 2 回押す。
 		</Japanese>
                 <Chinesesimp>
-			如何在远程遥控载具内的某一单位：
-			%1 1) 在右侧编辑列表选中单位。
-			%1 2) 载具上将会弹出对应目标单位的新图标。=
-			%1 3) 按下%3[左CTRL]%4同时在单位图标上双击%3[鼠标右键]%4。
-		</Chinesesimp>
+   如何远程遥控载具内的某一单位：
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 7]
```diff
@@ -5529,11 +6165,17 @@
...
+   %1 1) 在左侧编辑列表选中单位。
+   %1 2) 载具上将会弹出对应目标单位的新图标。
+   %1 3) 按下%3[左CTRL]%4同时在单位图标上双击%3[鼠标右键]%4。
+  </Chinesesimp>
+                <Chinese>
+   如何遠程遙控載具內的某一單位：
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 8]
```diff
@@ -5560,10 +6203,15 @@
...
 		</Japanese>
                 <Chinesesimp>
-			&lt;t size='6' shadow='0'&gt;&lt;img image='\achilles\data_f_achilles\icons\icon_achilles_hint.paa' /&gt;&lt;/t&gt; &lt;br /&gt;
-			最新功能： &lt;br /&gt;
-			&lt;t size='0.5' &gt;&lt;img image='A3\Ui_f\data\IGUI\RscIngameUI\RscHint\indent_square' /&gt;&lt;/t&gt;功能 &lt;br /&gt;
-		</Chinesesimp>
+   &lt;t size='6' shadow='0'&gt;&lt;img image='\achilles\data_f_achilles\icons\icon_achilles_hint.paa' /&gt;&lt;/t&gt; &lt;br /&gt;
+   最新功能： &lt;br /&gt;
+   &lt;t size='0.5' &gt;&lt;img image='A3\Ui_f\data\IGUI\RscIngameUI\RscHint\indent_square' /&gt;&lt;/t&gt;特色 &lt;br /&gt;
+  </Chinesesimp>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation

File: @AresModAchillesExpansion/addons/language_f/stringtable.xml [hunk 9]
```diff
@@ -5560,10 +6203,15 @@
...
+                <Chinese>
+   &lt;t size='6' shadow='0'&gt;&lt;img image='\achilles\data_f_achilles\icons\icon_achilles_hint.paa' /&gt;&lt;/t&gt; &lt;br /&gt;
+   最新功能： &lt;br /&gt;
+   &lt;t size='0.5' &gt;&lt;img image='A3\Ui_f\data\IGUI\RscIngameUI\RscHint\indent_square' /&gt;&lt;/t&gt;特色 &lt;br /&gt;
+  </Chinese>
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/language_f/stringtable.xml]: Indentation

**Comment by GodofMonkeys** [comment]
@CreepPork Ok, it's done. 👍

**Pull request merged**